tests/e2e/cloud_test.go

```diff
@@ -224,88 +224,3 @@ func TestRouteNetworksPodIPsAreAccessible(t *testing.T) {
 		t.Fatal(err)
 	}
 }
-
-func TestRouteDeleteCorrectRoutes(t *testing.T) {
```

Comment:
Why are we removing this?

Reply:
One idea for this test:

1. Manually create Routes for a made up server
2. Create `Node` object for made up node, with provider id and pod cidr set
3. HCCM will reconcile the node and delete it
4. All the events to nodes will trigger a reconcile of the routes.